app/data/data.json

```diff
@@ -39,6 +39,7 @@
           { "name": "subgrid", "id": "PiECNdst", "tags": ["VALUE", "NEW"] }] },
         { "name": "position", "id": "s1SmxA4X", "tags": ["PROP"], "children": [
           { "name": "static", "id": "SFXkEMHw", "tags": ["VALUE"] },
+          { "name": "sticky", "id": "KFxr1QE0", "tags": ["VALUE"] },
```

Comment:
Can you please add the `NEW` tag (like in `subgrid` a few lines up) since support is quite low.

http://caniuse.com/#feat=css-sticky